src/style.css

```
@@ -1,3 +1,68 @@
+:root {
+  --main-color: #00ACDC;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
```

```suggestion
.nav {
```
Consider using class selectors for some specific styling